addons/website/static/src/components/views/page_views_mixin.js

```diff
@@ -73,6 +97,7 @@ export const PageControllerMixin = (component) => class extends component {
 
     onSelectWebsite(website) {
         this.state.activeWebsite = website;
+        this.state.firstLoad = false;
```

Comment:
Seems like a bandage to hide the issue rather than a real fix imo but I'll wait for your reply on other points before going further and judging this new param